pkg/operators/ebpf/ebpf.go

```diff
@@ -641,6 +641,10 @@ func (i *ebpfInstance) Start(gadgetCtx operators.GadgetContext) error {
 	}
 	i.collection = collection
 
+	for name, m := range i.collection.Maps {
+		gadgetCtx.SetVar(name, m)
```

Comment:
When I said namespace, I meant something simple like using `"map/'` + mapName` to avoid maps clashing with other objects. Maps clashing with maps is something more difficult to support now. related to supporting multiple ebpf instances as well.